measures/measures-schema.yaml

```diff
@@ -197,3 +205,55 @@ definitions:
       - thoracicSurgery
       - urology
       - vascularSurgery
+
+  eligibilityOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
+      sexCode:
+        enum: [M, F]
+      minAge:
+        type: integer
+      maxAge:
+        type: integer
+      diagnosisCodes: { $ref: #/definitions/arrayOfStringIdentifiers }
+      additionalDiagnosisCodes: { $ref: #/definitions/arrayOfStringIdentifiers }
+      procedureCodes:
+        type: array
+        items: { $ref: #/definitions/codeObject }
+    required: [submissionMethods]
+
+  performanceOption:
+    type: object
+    properties:
+      submissionMethods: { $ref: #/definitions/qualityCodesSubmissionMethods }
+      optionType:
+        enum: [performanceMet, performanceNotMet, performanceExclusion]
```

Comment:
yep, both are PD_X for us. From Manoj:
"For C2Q we have confirmed from PQMM that there is no differentiation between exclusion and exception the way we (C2Q) calculate measures. I am not sure if that’s the case for non-claim measures. In multiple meetings, as recent as on 05/18 walkthrough we have heard from the program that “exclusion” is different than “exception” and should be captured separately." 

which seems pretty ambiguous to me about whether they should be presented independently in this json. We definitely do not need them differentiated for our calculations, and they are definitely not specified explicitly in the single source excel doc.